test/tests/parser/active_inactive/tests

```diff
@@ -0,0 +1,29 @@
+[Tests]
+  [./inactive_active]
+    type = Exodiff
+    input = 'active_inactive.i'
+    exodiff = 'active_inactive_out.e'
+    cli_args = 'Outputs/exodus=true'
+  [../]
+
+  [./inactive_section_missing]
+    type = RunException
+    input = 'active_inactive.i'
+    cli_args = 'AuxKernels/active=aux5'
+    expect_err = 'One or more active/inactive lists in the input file are missing a referenced section'
+  [../]
+
+  [./active_section_missing]
+    type = RunException
+    input = 'active_inactive.i'
+    cli_args = 'Kernels/inactive=convection BCs/inactive=top'
```

Comment:
Why are you passing `BCs/inactive=top`? Just the Kernel part should be enough to trigger the error, right?